tensorflow/tools/tf_sig_build_dockerfiles/manylinux_build.sh

```diff
@@ -0,0 +1,222 @@
+
+# Add the ROCm package repo location
+echo -e "[ROCm]\nname=ROCm\nbaseurl=$RPM_ROCM_REPO\nenabled=1\ngpgcheck=0" >>/etc/yum.repos.d/rocm.repo
+echo -e "[amdgpu]\nname=amdgpu\nbaseurl=https://repo.radeon.com/amdgpu/latest/rhel/7.9/main/x86_64/\nenabled=1\ngpgcheck=0" >>/etc/yum.repos.d/amdgpu.repo
+
+# Install the ROCm rpms
+yum clean all
+yum install -y libdrm-amdgpu
+yum install -y rocm-dev
+# yum install -y miopen-hip miopengemm rocblas rocrand rocfft hipfft hipblas rocprim hipcub rccl hipsparse
+yum install -y miopen-hip miopen-hip-devel miopengemm rocblas rocblas-devel rocsolver-devel rocrand-devel rocfft-devel hipfft-devel hipblas-devel rocprim-devel hipcub-devel rccl-devel hipsparse-devel hipsolver-devel
+
+# Ensure the ROCm target list is set up
+bash -c 'echo -e "gfx900\ngfx906\ngfx908\ngfx90a\ngfx1030" >> $ROCM_PATH/bin/target.lst'
+
+#Clone and install Tensorflow with rocm
+cd $BUILD_DIR
+if [ "$TF_VERSION" = "nightly" ]; then
+    export TF_ROCM_GCC=1
+    git clone https://github.com/ROCmSoftwarePlatform/tensorflow-upstream tensorflow
+    # build:rbe_linux_rocm_base --action_env=TF_ROCM_GCC=1
+    REF_LINE='build:rbe_linux_rocm_base --action_env=TF_ROCM_CONFIG_REPO="@ubuntu20.04-gcc9_manylinux2014-rocm_config_rocm"'
+    sed -i "s/$REF_LINE/$REF_LINE\nbuild:rbe_linux_rocm_base --action_env=TF_ROCM_GCC=1/" $BUILD_DIR/tensorflow/.bazelrc
+    cat $BUILD_DIR/tensorflow/.bazelrc
+else
+    git clone --branch r${TF_VERSION}-rocm-enhanced https://github.com/ROCmSoftwarePlatform/tensorflow-upstream tensorflow
+fi
+
+# Install Bazel
+BAZEL_VERSION=$(cat $BUILD_DIR/tensorflow/.bazelversion) # get the right version of bazel
+wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh &&
+    chmod -x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && bash bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+
+# export env variables
+export ROCM_PATH=$ROCM_PATH
+export GCC_HOST_COMPILER_PATH=/opt/rh/devtoolset-10/root/usr/bin/gcc
+export TF_PKG_LOC=/tmp/tensorflow_pkg
+export TF_CONFIGURE_IOS=0
+
+# First positional argument (if any) specifies the ROCM_INSTALL_DIR
+export ROCM_INSTALL_DIR=$ROCM_PATH
+export ROCM_TOOLKIT_PATH=$ROCM_INSTALL_DIR
+
+# build wheel
+cd $BUILD_DIR/tensorflow && yes "" | TF_NEED_ROCM=1 ROCM_TOOLKIT_PATH=${ROCM_INSTALL_DIR} PYTHON_BIN_PATH=${PYTHON_BIN_PATH} ./configure &&
+    bazel build --config=opt --action_env TF_ROCM_GCC=1 --config=rocm //tensorflow/tools/pip_package:build_pip_package --verbose_failures &&
+    bazel-bin/tensorflow/tools/pip_package/build_pip_package $TF_PKG_LOC --rocm --project_name tensorflow_rocm
+
+# check that wheels are manylinux compatiable
+TF_WHEEL=$(ls -Art $TF_PKG_LOC/tensorflow*.whl | tail -n 1)
+echo "Checking $TF_WHEEL..."
+pip3 install auditwheel
+time auditwheel repair --plat manylinux2014_x86_64 "$TF_WHEEL" --wheel-dir $TF_PKG_LOC 2>&1 | tee ${TF_WHEEL}_check.txt
```

Comment:
For some reason `auditwheel` repair is breaking or removing ROCm support. Let's change this to a simple filename swap. We'll just need to swap `linux` to `manylinux2014` like

```
TF_WHEEL_NEWNAME=${TF_WHEEL/linux/"manylinux2014"}
echo "rename whl to $TF_WHEEL_NEWNAME"
mv $TF_WHEEL $TF_WHEEL_NEWNAME
```

Reply:
Done.

Reply:
Will see how ci reacts and then cleanup